time stamp
float64[] data
string[] names
uint32 length
